modules/Books.js

```diff
@@ -0,0 +1,68 @@
+// Form local storage availability checker function
+export function isStorageAvailable(type) {
+  let storage;
```

Comment:
- [ ] You have done a great job using es6 syntax, the only thing you need to change is using arrow syntax for `isStorageAvailable` function 👍🏻

Reply:
Resolved. Thank you for your recommendation!